# HG changeset patch
# User Rik <rik@octave.org>
# Date 1500589778 25200
#      Thu Jul 20 15:29:38 2017 -0700
# Node ID a0b7a29338d558154e4df4f51cfc8799843fbd38
# Parent  7b43a96c21799981c735252f66524bd77bcdb718
Remove assert() accidentally left in cset 7b43a96c2179.

* oct-locbuf.h: Remove assert(0) from definition of OCTAVE_LOCAL_BUFFER
for C++14.

diff --git a/liboctave/util/oct-locbuf.h b/liboctave/util/oct-locbuf.h
--- a/liboctave/util/oct-locbuf.h
+++ b/liboctave/util/oct-locbuf.h
@@ -29,18 +29,17 @@ along with Octave; see the file COPYING.
 
 #include <algorithm>
 #include <memory>
 
 #if __cplusplus >= 201402L
 
 #define OCTAVE_LOCAL_BUFFER(T, buf, size)                               \
   auto octave_local_buffer_ ## buf = std::make_unique<T []> (size);     \
-  T *buf = octave_local_buffer_ ## buf.get (); \
-  assert (0);
+  T *buf = octave_local_buffer_ ## buf.get ()
 
 #else
 
 #define OCTAVE_LOCAL_BUFFER(T, buf, size)                               \
   std::unique_ptr<T []> octave_local_buffer_ ## buf { new T [size] };   \
   T *buf = octave_local_buffer_ ## buf.get ()
 
 #endif
